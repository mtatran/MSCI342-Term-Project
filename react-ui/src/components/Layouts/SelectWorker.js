import React, { Component, Fragment } from "react";
import SelectWorkerForm from "../SelectWorkerForm";

class SelectWorker extends Component {
  render() {
    return (
      <Fragment>
        {/* <Header/> */}
        <SelectWorkerForm user={this.props}/>
      </Fragment>
    );
  }
}
export default SelectWorker;


