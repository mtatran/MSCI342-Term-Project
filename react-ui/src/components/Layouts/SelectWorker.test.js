import React from "react";
import { create } from "react-test-renderer";
import SelectWorker from './SelectWorker';
import { BrowserRouter } from "react-router-dom";

describe('Profile Snapshot', () => {
  test('snapshot renders', () => {
    const component = create(<BrowserRouter>
      < SelectWorker/>
    </BrowserRouter>);
    expect(component.toJSON()).toMatchSnapshot();
  });

});
