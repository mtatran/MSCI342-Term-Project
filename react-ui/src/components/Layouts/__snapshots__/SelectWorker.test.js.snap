// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Snapshot snapshot renders 1`] = `
<div
  className="WorkerSelectForm-container"
  style={
    Object {
      "boxSizing": "border-box",
      "marginLeft": "auto",
      "marginRight": "auto",
      "maxWidth": 1540,
      "paddingLeft": 15,
      "paddingRight": 15,
      "position": "relative",
    }
  }
>
  <div>
    <p
      className="Title-Style"
    >
      Sorry, no service workers for your school were found
    </p>
  </div>
  <div
    style={
      Object {
        "alignItems": "normal",
        "display": "flex",
        "flexGrow": 0,
        "flexShrink": 0,
        "flexWrap": "wrap",
        "justifyContent": "flex-start",
        "marginLeft": -15,
        "marginRight": -15,
      }
    }
  >
    <div
      align="center"
      style={
        Object {
          "boxSizing": "border-box",
          "flexBasis": "100%",
          "flexGrow": 0,
          "flexShrink": 0,
          "left": "auto",
          "marginLeft": "0%",
          "maxWidth": "100%",
          "minHeight": 1,
          "paddingLeft": 15,
          "paddingRight": 15,
          "position": "relative",
          "right": "auto",
          "width": "100%",
        }
      }
    >
      <br />
      <div>
        <a
          href="/dashboard"
          onClick={[Function]}
        >
          Home
        </a>
      </div>
    </div>
  </div>
  <span
    style={
      Object {
        "clear": "both",
        "display": "table",
      }
    }
  />
</div>
`;
